src/pages/Groups/Group1/Screen2.tsx

```diff
@@ -1,7 +1,134 @@
 import React from 'react'
+import LeftOutlined from '@ant-design/icons/lib/icons/LeftOutlined'
+import UnorderedListOutlined from '@ant-design/icons/lib/icons/UnorderedListOutlined'
+import {
+  WeekContainer,
+  Time_container,
+  Timing_container,
+  Bookings_container,
+  IndividualBookings,
+  TopNav,
+  Header,
+  EventText,
+  EventTextWeekly,
+  TimeText,
+  LocationText,
+  LocationTextWeekly,
+  BotNav,
+  StyledButton,
+  ButtonContainer,
+  ToggleButton,
+  Calendar_container,
+  Day_container,
+  Days,
+  EventCard,
+} from './styles/Screen2.styled'
+import homeIconUnselected from '../../../assets/Group1/homeIconUnselected.svg'
+import facilitiesIconUnselected from '../../../assets/Group1/facilitiesIconUnselected.svg'
+import calenderIconSelected from '../../../assets/Group1/calenderIconSelected.svg'
+import profileIconUnselected from '../../../assets/Group1/profileIconUnselected.svg'
+import { useState } from 'react'
 
 const Screen2 = () => {
-  return <>this is screen 2</>
+  const [isDailyMode, setDailyMode] = useState(false)
+
+  return (
+    <>
+      {renderTopNav(isDailyMode)}
+
+      <Bookings_container>
+        <ButtonContainer>
+          <ToggleButton onClick={() => setDailyMode(!isDailyMode)}>
+            {isDailyMode ? 'Show Weekly' : 'Show Daily'}
+          </ToggleButton>
+        </ButtonContainer>
+        {isDailyMode ? renderDailySchedule() : renderWeeklySchedule()}
+      </Bookings_container>
+      {renderBotNav()}
+    </>
+  )
 }
 
+const renderTopNav = (isDailyMode: boolean) => (
+  <TopNav>
+    <LeftOutlined />
+    <Header>{isDailyMode ? 'Oct 8, Friday (Semester 1, Week 7)' : 'Semester 1, Week 7'}</Header>
+    <UnorderedListOutlined />
+  </TopNav>
+)
+const renderBotNav = () => (
+  <BotNav>
+    <StyledButton src={homeIconUnselected} />
+    <StyledButton src={facilitiesIconUnselected} />
+    <StyledButton src={calenderIconSelected} />
+    <StyledButton src={profileIconUnselected} />
+  </BotNav>
+)
+
+const renderDailySchedule = () =>
+  mockValues.map((e) => (
+    <IndividualBookings background={e.type} key={e.id}>
+      <EventText>{e.event}</EventText>
+      <TimeText>
+        {e.time.start} - {e.time.end}
```

Comment:
try to use http://tc39wiki.calculist.org/es6/template-strings/
 
 In this case, it's ok. but when you have complicated strings, string literals (backticks) makes code a lot more readable

Reply:
fixed in commit "b633ef9099261dc56d797b8bc08d096769891e48"